charts/spire/charts/spiffe-oidc-discovery-provider/values.yaml

```diff
@@ -88,10 +88,46 @@ livenessProbe:
 ## @param podAnnotations [object] Pod annotations for Spire OIDC discovery provider
 podAnnotations: {}
 
-insecureScheme:
-  ## @param insecureScheme.enabled Flag to enable insecure schema
-  enabled: false
+tls:
+  spire:
+    ## @param tls.spire.enabled Use spire to secure the oidc-discovery-provider
+    enabled: true
+
+  externalSecret:
+    ## @param tls.externalSecret.enabled Provide your own certificate/key via tls style Kubernetes Secret
+    enabled: false
+    ## @param tls.externalSecret.secretName Specify which secret to use
+    secretName: ""
 
+  certManager:
+    ## @param tls.certManager.enabled Use certificateManager to create the certificate
+    enabled: false
+    issuer:
+      ## @param tls.certManager.issuer.create Create an issuer to use to issue the certificate
+      create: true
+      acme:
+        ## @param tls.certManager.issuer.acme.email Must be set in order to register with LetsEncrypt. By setting, you agree to their Terms of Service
+        email: ""
+        ## @param tls.certManager.issuer.acme.server Server to use to get certificate. Defaults to LetsEncrypt
+        server: https://acme-v02.api.letsencrypt.org/directory
+        # Testing server: https://acme-staging-v02.api.letsencrypt.org/directory
```

Comment:
What is Testing server for?

Reply:
letsencrypt has two different instances. production and their testing instance. Production has very strict rules and if you misconfigure too badly, you can ban yourself. Testing is useful for some folks to make sure everything handshakes properly before switching to production. Though with cert-manager, I've never seen it misconfigured badly enough to ban on their prod server.

Reply:
staging is very useful when testing infrastructure automation. E.g. create destroy workflows during CI.